angular/src/app/containers/form/form.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Inject,
+  OnInit
+} from '@angular/core';
+import {
+  FormBuilder,
+  FormGroup,
+} from '@angular/forms';
+import { Store } from '@ngrx/store';
+import * as reducers from '../../reducers';
```

Comment:
Je hoeft toch niet alle reducers te importeren? Als je `Store` importeert kun je in `ngOnInit()` de juiste selecteren. Bijkomend voordeel is dat je explicieter aangeeft welke 'substore' je gebruikt.